go/runtime/bundle/manager.go

```diff
@@ -437,6 +474,64 @@ func (m *Manager) fetchBundle(url string) (string, error) {
 	return file.Name(), nil
 }
 
+func (m *Manager) cleanStaleBundles(runtimeID common.Namespace) {
+	m.mu.Lock()
+	active, ok := m.cleanupQueue[runtimeID]
```

Comment:
I already commented that the manager should probably not know about active version. But could expose a method to remove all versions up to `version` or `maxVersion`.